personal_finances/user/jwt_utils.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime, timedelta
+import os
+from typing import Any
+import boto3
+import jwt
+
+
+def _get_jwt_secret() -> str:
```

Comment:
These secrets don't change often, to avoid overloading the SSM database, and/or paying more in requests, we can cache it here.

Reply:
done